api/types.go

```diff
@@ -0,0 +1,47 @@
+package api
+
+type Context struct {
+	State DebuggerState `json:"state"`
```

Comment:
@pmorie @ironcladlou I'm not familiar with that pattern, haven't dug into how kubernetes uses it. Can you expand upon that?